docs/how_to/track_objects.md

```diff
@@ -148,7 +148,7 @@ enabling the continuous following of the object's motion path across different f
 
 Annotating the video with tracking IDs helps in distinguishing and following each object
 distinctly. With the
-[`sv.LabelAnnotator`](https://supervision.roboflow.com/annotators/#supervision.annotators.core.LabelAnnotator)
+[`sv.LabelAnnotator`](https://supervision.roboflow.com/latest/annotators/#supervision.annotators.core.LabelAnnotator)
```

Comment:
```suggestion
[`sv.LabelAnnotator`](latest/annotators.md/#supervision.annotators.core.LabelAnnotator)
```

Reply:
I think this is how we should refactor our links. This way, they always work in the context of the currently viewed version. At least that's what I meant.

Reply:
> 

I know but I did not deliberately because technically that link can be showed in IDE for docs check so I prefered URL as well. (IDE,code editors cannot handle that)

Reply:
I think we need to prioritize websites over code editors.